### PR TITLE
Using assertions in Unit tests

### LINKED ISSUE #43788: Using assertions and require for sig-scheduling
**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.):

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
FEATURE REQUEST

Using assert and require from https://github.com/kubernetes/kubernetes/tree/master/vendor/github.com/stretchr/testify can help us reduce the number of lines of code that we use for statements like err!=nil etc. Also, we can have negative test cases which can improve our code coverage using require. Some other SIG's have started using them.

### PR DESCRIPTION
**Pull request by lichen2013**
Using assertions:

1. cmd/kube-controller-manager/app/controller_manager_test.go
2. pkg/controller/bootstrap/jws_test.go
3. pkg/controller/cloud/node_controller_test.go
4. pkg/controller/controller_utils_test.go

Partially fix https://github.com/kubernetes/kubernetes/issues/43788

### CODE REVIEW

**Comment by lichen2013** [comment]
CLA should be done.

File: cmd/kube-controller-manager/app/controller_manager_test.go [hunk 1]
```diff
@@ -73,9 +75,7 @@ func TestIsControllerEnabled(t *testing.T) {
 
 	for _, tc := range tcs {
 		actual := IsControllerEnabled(tc.controllerName, sets.NewString(tc.disabledByDefaultControllers...), tc.controllers...)
-		if actual != tc.expected {
-			t.Errorf("%v: expected %v, got %v", tc.name, tc.expected, actual)
-		}
+		assert.Equal(t, actual, tc.expected, fmt.Sprintf("%v: expected %v, got %v", tc.name, tc.expected, actual))
```
Comment (feiskyer) [inline cmd/kube-controller-manager/app/controller_manager_test.go]: Seems wrong usage here, should change the position of expected and actual.

**Review by feiskyer — COMMENTED** [review]
same with other places.

File: cmd/kube-controller-manager/app/controller_manager_test.go (hunk 1 above)
Comment (lichen2013) [reply cmd/kube-controller-manager/app/controller_manager_test.go]: o, let me check. Thanks!

**Comment by resouer** [comment]
The change lgtm, please squash your commits.

But this need approve from @deads2k to see if worth to fix.

**Comment by deads2k** [comment]
> But this need approve from @deads2k to see if worth to fix.

I have no real opinion.

/approve no-issue

File: pkg/controller/bootstrap/jws_test.go [hunk 2]
```diff
@@ -26,42 +30,29 @@ const (
...
 func TestComputeDetachedSig(t *testing.T) {
 	sig, err := computeDetachedSig(content, id, secret)
-	if err != nil {
-		t.Errorf("Error when computing signature: %v", err)
-	}
-	if sig != "eyJhbGciOiJIUzI1NiIsImtpZCI6Impvc2h1YSJ9..VShe2taLd-YTrmWuRkcL_8QTNDHYxQIEBsAYYiIj1_8" {
-		t.Errorf("Wrong signature. Got: %v", sig)
-	}
+	assert.NoError(t, err, "Error when computing signature: %v", err)
+	assert.Equal(t, "eyJhbGciOiJIUzI1NiIsImtpZCI6Impvc2h1YSJ9..VShe2taLd-YTrmWuRkcL_8QTNDHYxQIEBsAYYiIj1_8", sig, "Wrong signature. Got: %v", sig)
```
Comment (resouer) [inline pkg/controller/bootstrap/jws_test.go]: It's too long, please make it shorter. Same with or other places.

Comment (lichen2013) [reply pkg/controller/bootstrap/jws_test.go]: Done

File: pkg/controller/cloud/node_controller_test.go [hunk 3]
```diff
@@ -557,21 +535,11 @@ func TestZoneInitialized(t *testing.T) {
...
-	}
-
-	if fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneFailureDomain] != "us-west-1a" {
-		t.Errorf("Node FailureDomain not correctly updated")
-	}
+	assert.Equal(t, 1, len(fnh.UpdatedNodes), "Node was not updated")
+	assert.Equal(t, "node0", fnh.UpdatedNodes[0].Name, "Node was not updated")
+	assert.Equal(t, 2, len(fnh.UpdatedNodes[0].ObjectMeta.Labels), "Node label for Region and Zone were not set")
+	assert.Equal(t, "us-west", fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneRegion], "Node Region not correctly updated")
+	assert.Equal(t, "us-west-1a", fnh.UpdatedNodes[0].ObjectMeta.Labels[kubeletapis.LabelZoneFailureDomain], "Node FailureDomain not correctly updated")
```
Comment (resouer) [inline pkg/controller/cloud/node_controller_test.go]: still some very long lines here and other files, please check the whole change

Comment (lichen2013) [reply pkg/controller/cloud/node_controller_test.go]: Done. And checked other places too, I believe all assert lines can be viewed in one line now. Thanks!!

**Pull request merged**